ocfweb/docs/docs/services/web/flask.md

```diff
@@ -23,13 +23,21 @@ OCF servers requires only just a little extra configuration.
 
 * run.fcgi:
 
-      #!/usr/bin/env python
+      #!venv/bin/python
       from flup.server.fcgi import WSGIServer
       from flaskyapplication import app
 
       if __name__ == '__main__':
           WSGIServer(app).run()
 
+1.     Make a virtualenv.  This will allow you to *pip install* any python packages
+that you need that the OCF does not already have:
+```
+    $ virtualenv venv
```

Comment:
i'd suggest `virtualenv -p python3.5 venv`

hopefully most people are starting new things on python 3 (seems likely since that's what 61A teaches), and if not, this at least makes it obvious how to choose a different python version